src/routes.js

```diff
@@ -12,13 +12,14 @@ import NotFound from './views/NotFound';
 
 const page = View => () => <Layout><View /></Layout>;
 
-const entryPage = View => () => <LayoutEntry><View /></LayoutEntry>;
+// TODO: this call gives react warning about "unique key propety"
+const entryPage = components => () => <LayoutEntry>{components}</LayoutEntry>;
```

Comment:
This is a weird definition. I'm not sure how it works to pass components like that.